staging/src/k8s.io/apiextensions-apiserver/test/integration/ratcheting_test.go

```diff
@@ -219,6 +232,17 @@ func (u updateMyCRDV1Beta1Schema) Do(ctx *ratchetingTestContext) error {
 			sch.Properties = map[string]apiextensionsv1.JSONSchemaProps{}
 		}
```

Comment:
This block belongs to the code starting with `uuidString := string(uuid.NewUUID())`, at least the go doc suggests so